designer/client/components/Menu/Menu.tsx

```diff
@@ -86,19 +84,19 @@ export default function Menu({ updateDownloadedAt, id }: Props) {
 
       {page.isVisible && (
         <Flyout title="Add Page" onHide={page.hide}>
-          <PageCreate data={data} onCreate={() => page.hide()} />
+          <PageCreate onCreate={() => page.hide()} />
```

Comment:
why did we decide to remove passing the data object?

Reply:
I believe that change was already made from a previous commit [some menu items are using data as prop](https://github.com/XGovFormBuilder/digital-form-builder/pull/747/files)

Reply:
yup - should be passed in as context. prop drilling is annoying! and using context will keep it up to date